meson.build

```diff
@@ -15,6 +17,29 @@ sources = [
   'src/vapoursynth.cpp'
 ]
 
+libs += static_library('p2p_main',
+    [
+        'libp2p/p2p_api.cpp',
+        'libp2p/v210.cpp',
+        'libp2p/simd/cpuinfo_x86.cpp',
+        'libp2p/simd/p2p_simd.cpp'
+    ],
+    include_directories: ['libp2p', 'libp2p/simd'],
+    cpp_args: ['-std=c++14']
+)
+
+if host_machine.cpu_family().startswith('x86')
+    libs += static_library('p2p_sse41',
+        'libp2p/simd/p2p_sse41.cpp',
+        include_directories: ['libp2p', 'libp2p/simd'],
+        cpp_args: [
+          '-std=c++14',
+          '-DP2P_SIMD',
+          '-msse4.1'
```

Comment:
There's already MSVC project files for Windows, so I didn't consider that at all. I just want to compile the thing on my Manjaro installation. Also never heard of simd module, will read those docs later after work.

Reply:
Windows support is indeed pointless. You can link gcc/clang FFmpeg if you want and that's where the speed ifference would come from anyway.

Reply:
Using flags incompatible with MSVC still means that Windows users get a compilation failure instead of either a fast *or* slow version of the built project.

One of the advantages of Meson is that it's cross-platform *without* requiring handwritten XML, so there are advantages to making sure that stays functional.